packages/hw-app-eth/src/Eth.js

```diff
@@ -22,6 +22,9 @@ import { EthAppPleaseEnableContractData } from "@ledgerhq/errors";
 import type Transport from "@ledgerhq/hw-transport";
 import { BigNumber } from "bignumber.js";
 import { encode, decode } from "rlp";
```

Comment:
Since we switch to ethers, we should drop rlp and use ethers.utils.RLP.
However I tried and it's not a dropin replacement. Shoud be similar though.

Reply:
done